README.md

```diff
@@ -173,46 +192,98 @@ To build the modern eBPF probe, you need:
     ```
 * A kernel version >=`5.8`.
 
-Then, issue:
+> __NOTE:__ These are not the requirements to use the modern BPF probe, but rather for building it from source.
```

Comment:
@Andreagit97 are these changes ok? Would you have additional improvement suggestions? Thanks in advance!

Reply:
Super detailed, thanks :+1: !